cypress/integration/select-dataset.spec.ts

```diff
@@ -0,0 +1,14 @@
+import { datasetPage } from "../page-objects/app/datasets-page";
+
+describe("XL Dataset should be opened in decision tree", () => {
+
+    it("should click on XL dataset and open it in decision tree", () => {
```

Comment:
About the comments:
1. Inline comments looks bad and it's hard to read. Please place them on the previous line.
2. I'd say they are redundant, because it's easy to read the same from code.